README.md

```diff
@@ -44,10 +49,35 @@ DataLens consists of the three main parts:
+We are releasing DataLens with first minimal set of available connectors (clickhouse, clickhouse over ytsaurus and postgresql) as well as other core functionality such as data processing engine and user interface. However, to kick off this project in a reasonable timeframe we have chosen to drop some of the features out of the first release: this version does not contain middleware and components for user sessions, object ACLs and multitenancy (although code contains entry-points for such extensions). We are planning to add missing features based on our understanding of community priorities and your feedback.
 
 ## FAQ
 
-**Where can I find persistent application data storage?**
+**Where does DataLens store it's metadata?**
+
+We use the `metadata` folder to store PostgreSQL data. If you want to start over, you can delete this folder: it will be recreated with demo objects on the next start of the `datalens-us` container.
+
+**I use the `METADATA_POSTGRES_DSN_LIST` param for external metadata database and app does not start. What could be the reason?**
+
+We use some PostgresSQL extensions for the metadata database and the application checks for them on startup and tries to install them if they are not already installed. Check your database user's rights for installing extensions by trying to install them manually:
+
+```sql
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
+CREATE EXTENSION IF NOT EXISTS btree_gin;
+CREATE EXTENSION IF NOT EXISTS btree_gist;
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+```
+
+If this attempt is unsuccessful, try to install dependencies by database admin and add param `METADATA_SKIP_INSTALL_DB_EXTENSIONS=1` on startup, this parameter allows app to skip installing extensions.
+
+If you're using managed database it's also possible that extensions for your database cluster are controlled by external system and could be changed only using it's UI or API. In that case, consult with documentation for managed database service that you're using. Don't forget to add `METADATA_SKIP_INSTALL_DB_EXTENSIONS=1` after installing extensions this way.
+
+**My PostgresSQL cluster has multiple hosts, how can I specify them in `METADATA_POSTGRES_DSN_LIST` param?**
+
+Yes, you can write all cluster hosts separated by commas:
+
+`METADATA_POSTGRES_DSN_LIST="postgres://{user}:{password}@{host_1}:{port}/{database},postgres://{user}:{password}@{host_2}:{port}/{database},postgres://{user}:{password}@{host_3}:{port}/{database}" ...`
+
+**How can I specify custom certificate for connecting to metadata database?**
 
-We use the `.us-data` folder to store PostgreSQL data permanently. You can delete this folder if you want, it will be recreated with the demo data after restarting the `datalens-us` container.
\ No newline at end of file
+You can add additional certificates for database to `./certs/root.crt`, they will be used for connecting to database from the `datalens-us` container.
```

Comment:
`for database to` -> it's better `to database in`
`they will be used for connecting to database` -> `they will be used to connect to the database` or `they will be used for connection[s] to the database`

Reply:
fixed